requirements.txt

```diff
@@ -16,3 +16,4 @@ shellcheck
 scrypt
 license-expression
 setuptools_scm >=7
+cirun
```

Comment:
In https://github.com/conda-forge/cf-autotick-bot-test-package-feedstock/pull/446 we need to install cirun from `main`. Is that the case here too?

Reply:
latest version is already released, so this is fine to use